### ISSUE TITLE
Proposal: New Failure Technique — Failure of SC 2.1.1 and SC 3.2.2 due to using `keydown`/`keyup` event handlers without guarding against IME composition

### ISSUE THREAD

**Issue by w3cbot** [w3c/i18n-activity#2121]
**This is a tracker issue.** Only discuss things here if they are i18n group internal meta-discussions about the issue. **Contribute to the actual discussion at the following link:**

§ https://github.com/w3c/wcag/issues/5075
